testdata/baselines/reference/submoduleAccepted/conformance/jsDeclarationsUniqueSymbolUsage.types.diff

```diff
@@ -1,15 +1,13 @@
 --- old.jsDeclarationsUniqueSymbolUsage.types
 +++ new.jsDeclarationsUniqueSymbolUsage.types
-@@= skipped -15, +15 lines =@@
-  * @param {import('./a').WithSymbol} value 
+@@= skipped -16, +16 lines =@@
   */
  export function b(value) {
-->b : (value: import("./a").WithSymbol) => import("./a").WithSymbol
+ >b : (value: import("./a").WithSymbol) => import("./a").WithSymbol
 ->value : import("a").WithSymbol
-+>b : (value: any) => any
-+>value : any
++>value : import("./a").WithSymbol
 
      return value;
 ->value : import("a").WithSymbol
-+>value : any
++>value : import("./a").WithSymbol
```

Comment:
interestingly, this seems buggy in Strada - but this is already in the `submoduleAccepted` here so 🤷 

https://github.com/microsoft/TypeScript/blob/e6a50a78619632bba7af4f23b2e7e4940b9cf788/tests/baselines/reference/jsDeclarationsUniqueSymbolUsage.types#L18-L31